interface/vmck.py

```diff
@@ -34,21 +34,24 @@ def evaluate(submission):
     options = vmck_config(submission)
     name = f'{submission.assignment.full_code} submission #{submission.id}'
     options['name'] = name
-    options['manager'] = True
+    options['restrict_network'] = True
+
+    options['manager'] = {}
+    options['manager']['vagrant_tag'] = settings.MANAGER_TAG
+    options['manager']['memory'] = settings.MANAGER_MEMORY
```

Comment:
Q: Maybe we should also add here, ```MB``` (to have the measurement unit as we do for the CPU?)

Reply:
You mean as a comment or change the name of the option (i. e. `memory_MB`)?

Reply:
As the name of the option